torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1144,7 +1146,8 @@ def evaluate_expr(self, expr: "sympy.Expr"):
         """
         Given an expression, evaluates it, adding guards if necessary
         """
-        if len(expr.free_symbols) == 0:
+        # NB: FloorDiv currently needs to be evaluated in _maybe_evaluate_static
+        if not expr.has(FloorDiv) and len(expr.free_symbols) == 0:
```

Comment:
Needed to eval Float FloorDiv exprs since we only have a dedicated rule for Integers